src/pages/fileupload/FileUpload.mdx

```diff
@@ -0,0 +1,85 @@
+---
+name: File Upload
+route: /fileupload
+---
+
+# File Upload
+
+
+import FileUpload from './FileUpload.state';
+
+## Basic
+<FileUpload />
+
+
+```jsx
+<FileUpload
+  acceptedFiles={[
+    "png",
+    "jpg",
+    "jpeg",
+    "pdf"
+  ]}
+  files={[
+    {
+      name: 'https://cdn.synapsepay.com/uploads/2019/10/02/3BmuL7.png'
+    }
+  ]}
+  fileValues={{
+    'bd58154cd50dab5064b577e470f52474f0b57aa2aa33ed18d616f551b307ee03': {
+      'subDocumentId': 'bd58154cd50dab5064b577e470f52474f0b57aa2aa33ed18d616f551b307ee03',
+      'documentValue': 'https://cdn.synapsepay.com/uploads/2019/10/02/3BmuL7.png'
+    }
+  }}
+  deleteAction={() => console.log('Delete Action')}
```

Comment:
For the purpose of demonstrating the functionality of `deleteAction` and `onDrop` lets have these be alerts instead. Something like "This is the deleteAction which would delete the file" and "This is the onDrop function, which would be a function that updates the files".

Reply:
Updated functions use alerts instead of console log